vnext/package.json

```diff
@@ -69,5 +69,51 @@
       "minor",
       "patch"
     ]
-  }
+  },
+  "files": [
+    "/codegen",
+    "/Chakra",
+    "/Common",
+    "/DeforkingPatches",
+    "/etc",
+    "/flow",
+    "/Folly",
+    "/include",
+    "/JSI",
+    "/Libraries",
+    "!/Libraries/**/*.windesktop.*",
+    "/local-cli",
+    "/Microsoft.ReactNative",
+    "/Microsoft.ReactNative.Cxx",
+    "/Microsoft.ReactNative.Managed",
+    "/Microsoft.ReactNative.SharedManaged",
+    "/Mso",
+    "/PropertySheets",
+    "/ReactCommon",
+    "/ReactUWP",
+    "/ReactWindowsCore",
+    "/Scripts",
+    "/Shared",
+    "/stubs",
+
+    "/.flowconfig",
+    "/Directory.Build.props",
+    "/Directory.Build.targets",
+    "/index.js",
+    "/index.windows.d.ts",
+    "/index.windows.d.ts.map",
+    "/index.windows.js",
+    "/index.windows.js.map",
+    "/interface.js",
+    "/just-task.js",
```

Comment:
This one isn't needed. - its internal build logic.

Reply:
#4813